### PR TITLE
GLUE-10222 MerchantsRestApi integration guide.

### PR DESCRIPTION
**Pull request by az-joss**
## PR Description

Integration guide for marketplace merchant api.

Ticket: https://spryker.atlassian.net/browse/GLUE-10222

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202108.0/glue/marketplace-merchant-feature-integration.md [hunk 1]
```diff
@@ -57,13 +57,19 @@ Make sure that the following changes have been applied in transfer objects:
 | RestOrdersAttributesTransfer.merchantReferences | property | Created | src/Generated/Shared/Transfer/RestOrdersAttributesTransfer |
 | RestOrderDetailsAttributesTransfer.merchantReferences | property | Created | src/Generated/Shared/Transfer/RestOrderDetailsAttributesTransfer |
 | RestOrderItemsAttributesTransfer.merchantReference | property | Created | src/Generated/Shared/Transfer/RestOrderItemsAttributesTransfer |
-| RestErrorMessageTransfer | object | Created | src/Generated/Shared/Transfer/RestErrorMessageTransfer |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/glue/marketplace-merchant-feature-integration.md:60]: why was it removed? Features which this one depends on already contain it?

**Review by ilyakubanov — COMMENTED** [review]
Please check the corresponding PR: https://github.com/spryker/suite-nonsplit/pull/3899/files , not all the plugins are added (mention MerchantReferencesOrderExpanderPlugin as well in the corresponding IG). 
Please also recheck verification steps to contain all the provided by this IG logic.

**Comment by ilyakubanov** [comment]
please also check why CI is red

File: docs/marketplace/dev/feature-integration-guides/202108.0/glue/marketplace-merchant-feature-integration.md (hunk 1 above)
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202108.0/glue/marketplace-merchant-feature-integration.md:60]: is supposed, that `RestErrorMessageTransfer ` is same for any rest module. And this one comes from Rest API Core (glue) right?
@ilyakubanov

**Comment by az-joss** [comment]
@ilyakubanov
Some plugins on that particular ticket has own feature repo and feature integration:

**[Merchant Opening hours REST](https://release.spryker.com/modules/view/4925)**
`MerchantOpeningHoursByMerchantReferenceResourceRelationshipPlugin`
`MerchantOpeningHoursResourceRoutePlugin`

**[Merchant Order Feature](https://release.spryker.com/features/view/115)**
`MerchantReferencesOrderExpanderPlugin`

**[Basic] [Order Management Feature](https://release.spryker.com/features/view/58)**
`OmsStatesOrderExpanderPlugin`

**[Basic] [Order Management REST API Feature](https://release.spryker.com/modules/view/4927)**
`OmsRestOrderItemsAttributesMapperPlugin`

`[Basic]` - it is guides from basic core that seems in the other repository. And it is still not updated unfortunately.

**Review by ilyakubanov — APPROVED** [review]

**Pull request merged**